src/test/java/Homework21.java

```diff
@@ -1,40 +1,19 @@
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.annotations.Test;
 
 public class Homework21 extends BaseTest{
-
+String playListName= "Ali";
 @Test
-    public static void renamePlaylist() throws InterruptedException {
```

Comment:
**Please, delete** `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://user-images.githubusercontent.com/104761482/212845959-e200f64c-9cc5-4569-bd37-40cef6608cb5.png)

Reply:
Yea i''l stop for sure. Thanks for the feedback